leader-election-impl/src/main/java/com/palantir/leader/proxy/AwaitingLeadershipProxy.java

```diff
@@ -214,7 +218,8 @@ protected Object handleInvocation(Object proxy, Method method, Object[] args) th
         Object maybeValidDelegate = delegateRef.get();
 
         ListenableFuture<StillLeadingStatus> leadingFuture =
-                statusRetrier.execute(() -> leaderElectionService.isStillLeading(leadershipToken));
+                Tracers.wrapListenableFuture("validate-leadership",
```

Comment:
do we want one on each invocation that it retries?

Reply:
this would still remain as the overall trace for the `validate-leadership` portion, but I imagine that each invocation inside `statusRetrier` should be wrapped in one where it's like `validate-leadership attempt 0` etc.

Reply:
fair enough, not strictly necessary tbf